make/modules/java.base/Java.gmk

```diff
@@ -34,6 +34,7 @@ EXCLUDE_FILES += \
   $(TOPDIR)/src/java.base/share/classes/jdk/internal/module/ModuleLoaderMap.java
 
 EXCLUDES += java/lang/doc-files
+EXCLUDES += jdk/classfile/snippets
```

Comment:
I don't like hard-coded excludes like this. Is this the first real-world example of snippets in the JDK? If not, how has this been resolved in other instances? Maybe we need a more general method to exclude snippets from compiling.

Reply:
Unfortunately I found only two use cases of external snippets in JDK. One is explicitly excluding the snippets package and the other does not care, so the snippets are included.

Reply:
Is the practice of naming the directory `snippets` well-established or mandated? Otherwise I think a name like `snippet-files` (akin to `doc-files`) would be much better, since it indicates that is not a java package name. (These do not usually include `-`; I am unsure if this is prohibited by the specification, or just not used by convention.)

In fact, when checking, I see that there is already a handful of `snippet-files`:
```
./jdk.jfr/share/classes/jdk/jfr/snippet-files
./jdk.jfr/share/classes/jdk/jfr/consumer/snippet-files
./java.compiler/share/classes/javax/tools/snippet-files
```

So maybe rename it to `snippet-files`, and I'll look into making exclusion of `snippet-files` be done automatically.

Reply:
It would be nice to get the directory excluded by default. I used snippet-files name since it was mentioned in the JEP https://openjdk.org/jeps/413

Reply:
Snippets moved from jdk/classfile/snippets to jdk/classfile/snippet-files and explicit exclusion of will be removed when excluded by default.
Thanks!